### PR TITLE
chore: Removed button prop from actionbar and detailsidebar component

### PR DESCRIPTION
**Pull request by anikethsaha**
**Type:**

The following has been addressed in the PR:

- There is a related issue? NA
- Unit or Functional tests are included in the PR ? Yes

**Description:**
Changed the hover effect of the actionbar and detailsidebar

#### Current

image

#### New changes

image

Update
---
I have removed the `button` prop from these components.

### CODE REVIEW

File: src/components/ActionBar/ActionBar.tsx [hunk 1]
```diff
@@ -119,7 +120,15 @@ class ActionBar extends Component {
     if (renderList.length > 0) {
       return (
         <List>
-          <ActionListItem alignItems={'flex-start'} button={true}>
+          <ActionListItem
+            alignItems={'flex-start'}
+            button={true}
+            className={css`
```
Comment (ayusharma) [inline src/components/ActionBar/ActionBar.tsx]: Please do not use inline css.

File: src/components/DetailSidebar/DetailSidebar.tsx [hunk 2]
```diff
@@ -41,7 +42,15 @@ const renderTitle = (packageName, packageVersion, packageMeta): JSX.Element => {
 
   return (
     <List className="detail-info">
-      <TitleListItem alignItems="flex-start" button={true}>
+      <TitleListItem
+        alignItems="flex-start"
+        button={true}
+        className={css`
```
Comment (ayusharma) [inline src/components/DetailSidebar/DetailSidebar.tsx]: same here.. Inline css.

File: src/components/DetailSidebar/styles.ts [hunk 3]
```diff
@@ -3,11 +3,12 @@ import ListItemText from '@material-ui/core/ListItemText';
 
 import ListItem from '../../muiComponents/ListItem';
 
-export const TitleListItem = styled(ListItem)({
-  paddingLeft: 0,
-  paddingRight: 0,
-  paddingBottom: 0,
-});
+export const TitleListItem = styled(ListItem)(`
+padding: 0 0.5rem;
```
Comment (ayusharma) [inline src/components/DetailSidebar/styles.ts]: Why are you using `rem` ? The rem unit means "The root element's font-size".

Comment (anikethsaha) [reply src/components/DetailSidebar/styles.ts]: Yea, wanted to make it relative with its roots! Should I change it to px?

Comment (ayusharma) [reply src/components/DetailSidebar/styles.ts]: There is no need for styling now. We have to remove the hover effect (grey background color). you can remove button={true}. Please see the review summary.

Comment (anikethsaha) [reply src/components/DetailSidebar/styles.ts]: You want me to remove hover styling and `button={true}` from both the component?

Comment (juanpicado) [reply src/components/DetailSidebar/styles.ts]: yeah, I think the button hover effect was a mistake from previous migrations. Actually mine.

Comment (anikethsaha) [reply src/components/DetailSidebar/styles.ts]: ok will remove it by next week, a bit busy with my college exams !

Comment (anikethsaha) [reply src/components/DetailSidebar/styles.ts]: done 👍

Comment (juanpicado) [reply src/components/DetailSidebar/styles.ts]: @anikethsaha again the snapshots :(

Comment (anikethsaha) [reply src/components/DetailSidebar/styles.ts]: I did update the snapshots after removing button prop

**Comment by priscilawebdev** [comment]
Hi @anikethsaha ! it seems that you need to fix some tests. Please see:

image

**Comment by priscilawebdev** [comment]
please @anikethsaha do you have any updates on this PR?

**Comment by anikethsaha** [comment]
I think I cant work on this for next one week. 
If anyone wants to take up can go ahead 🙂

**Review by valeriiStarykEleks — APPROVED** [review]

**Comment by juanpicado** [comment]
Closing in favor of the new location for improvements, here https://github.com/verdaccio/verdaccio/tree/master/packages/plugins/ui-theme

**Pull request closed without merging**